met/src/basic/vx_util/Makefile.am

```
@@ -69,6 +69,7 @@ libvx_util_a_SOURCES = ascii_table.cc ascii_table.h \
                GridOffset.h GridOffset.cc \
                observation.h observation.cc \
                stat_column_defs.h \
+	       handle_openmp.h handle_openmp.cc \
```

```suggestion
               handle_openmp.h handle_openmp.cc \
```
Just lining things up.